tests/test_mapping/test_mapping_helpers.py

```diff
@@ -267,11 +266,16 @@ class TestValidateMappingFilenames(object):
     def test_validate_mapper_config_raises_file_incorrect(self):
         config = {
             'years' : {'survey_year': 2022,},
-            '2022_mappers': {'itl_mapper_path': "itl_2022.cs",
-            'ultfoc_mapper_path': "BERD_2022_ultfoc.csv",},
-            '2023_mappers': {'itl_mapper_path': "itl_2023.csv",
-            'ultfoc_mapper_path': None,}
-            } 
+            '2022_mappers': {
```

Comment:
We need only one section for mappers in the user config, <this year>_mappers. I am happy to pass it for now but we need to decide what to do,

Reply:
you are right, we'll sort it out